android/src/main/java/io/didomi/reactnative/DidomiModule.kt

```diff
@@ -1179,7 +1179,7 @@ class DidomiModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     // Required to transform from array to variadic.
     private fun readableArrayToStringArray(readableArray: ReadableArray): Array<String> {
-        return Array(readableArray.size()) { i -> readableArray.getString(i) }
+        return Array(readableArray.size()) { i -> readableArray.getString(i) ?: "" }
```

Comment:
We can avoid unnecessary values here
```suggestion
        return readableArray.toSet().toTypedArray()
```

Reply:
or even
```
    private fun readableArrayToStringArray(readableArray: ReadableArray) = readableArray.toSet().toTypedArray()
```